packages/react-magma-dom/src/components/Select/index.tsx

```diff
@@ -116,6 +126,10 @@ export interface SelectProps<T extends SelectOptions>
    * Id of the element that describes the select trigger button
    */
   ariaDescribedBy?: string;
+  /**
+   * Positioning styles to apply to the floating element
+   */
+  floatingElementStyles?: React.CSSProperties;
```

Comment:
Can we add examples in storybook of how adopters would use this?

Reply:
These styles should be only for internal usage.
Marked it via @internal.